TeaDiary.domain/Models/User.cs

```diff
@@ -9,7 +9,7 @@ public class User: Resource
         public string Name { get; set; }
         public string PasswordHash { get; set; }
         public string EmailAdress { get; set; }
-        public IEnumerable<Tea> Teas { get; set; }
+        public IList<int> TeaIDs { get; set; }
```

Comment:
That's not how you model 'one to many' in DB. Your 'many' end should a foreign id (`tea.UserId`). Your 'one' end shouldn't have any foreign keys.

Reply:
Made `IEnumerable<tea>` back instead of `IList<int>`